src/sentry/api/endpoints/project_rules.py

```diff
@@ -103,6 +104,20 @@ def post(self, request, project):
             RuleActivity.objects.create(
                 rule=rule, user=request.user, type=RuleActivityType.CREATED.value
             )
+            for action in kwargs.get("actions"):
```

Comment:
Since this is repeated can you make it a function too?
```python
def create_alert_rule_actions(
    actions: Sequence[Mapping[str, str]], 
    rule: Rule, 
    request: Request,
) -> None:
    for action in actions:
        # Only call creator for Sentry Apps with UI Components for alert rules.
        if not action.get("hasSchemaFormConfig"):
            continue

        alert_rule_actions.AlertRuleActionCreator.run(
            install=SentryAppInstallation.objects.get(
                uuid=action.get("sentryAppInstallationUuid")
            ),
            fields=action.get("settings"),
            uri=action.get("uri"),
            rule=rule,
            request=request,
        )
```

Reply:
makes sense, will make the change in https://github.com/getsentry/sentry/pull/28659